src/google-calendar-api.js

```diff
@@ -57,17 +90,41 @@ module.exports = {
 			)
 		);
 	},
-	createWfhEvent: employeeName => {
+	createWfhEventInInterval: (employeeName, eventStartDateTime, eventEndDateTime) => {
```

Comment:
What will also help the caller not have to choose which method to call. They simply call one method with an optional parameter. It looked like the two were going the same thing other than formatting the dates being sent.